docs/conductor/getting-started.md

```diff
@@ -0,0 +1,87 @@
+# Getting started with Conductor
+##
+
+Conductor will group and schedule automated Dependency Updates on your own Continuous Integration platform. If the update succeeds, Conductor sends you a pull request to your code hosting platform (GitHub, GitLab, Bitbucket) with the changed composer.lock file and, if necessary, other files modified by Composer plugins or scripts.
```

Comment:
```suggestion
Conductor will group and schedule automated dependency updates on your own Continuous Integration platform. If the update succeeds, Conductor sends you a pull request to your code hosting platform (GitHub, GitLab, Bitbucket) with the changed composer.lock file and, if necessary, other files modified by Composer plugins or scripts.
```

Not sure how I looked at this sentence 2x already and didn't notice

Reply:
haha. even I had the diff and didnt see it!